projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.html

```diff
@@ -0,0 +1,13 @@
+<cdk-virtual-scroll-viewport appendOnly itemSize="80" class="tabs-viewport">
+    <div *cdkVirtualFor="let c of allColors; let i = index" class="grid-container">
+        <geoengine-color-attribute-input
+            class="grid-colorpicker"
+            [ngModel]="c"
+            (ngModelChange)="rebuildColorMap(i, $event)"
+        ></geoengine-color-attribute-input>
```

Comment:
```suggestion
    <div *cdkVirtualFor="let color of allColors; let i = index" class="grid-container">
        <geoengine-color-attribute-input
            class="grid-colorpicker"
            [ngModel]="color"
            (ngModelChange)="rebuildColorMap(i, $event)"
        ></geoengine-color-attribute-input>
```

Please use speaking names if possible instead of one-char stuff. Use these mainly for count variables like i, j, only.

Reply:
Done in [Margin for removebutton, improve comments](https://github.com/geo-engine/geoengine-ui/pull/614/commits/959e1fec8e9dbc3c99e72add43ed6d0bd294de48)